api/src/main/java/io/opentelemetry/metrics/LongUpDownCounter.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.metrics;
+
+import io.opentelemetry.metrics.LongUpDownCounter.BoundLongUpDownCounter;
+import java.util.Map;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * UpDownCounter instrument, to report measurements of long values. Cumulative values can go up or
+ * stay the same, but can never go down. Cumulative values cannot be negative.
+ *
+ * <p>Example:
+ *
+ * <pre>{@code
+ * class YourClass {
+ *   private static final Meter meter = OpenTelemetry.getMeterRegistry().get("my_library_name");
+ *   private static final LongCounter counter =
+ *       meter.
+ *           .longCounterBuilder("processed_jobs")
+ *           .setDescription("Processed jobs")
+ *           .setUnit("1")
+ *           .build();
+ *
+ *   // It is recommended that the API user keep a reference to a Bound Counter.
+ *   private static final BoundLongCounter someWorkBound =
```

Comment:
The example in here needs to be updated.

Reply:
Done.